pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -404,6 +408,9 @@
   front_join_us_challenge: "Win a video chat for your classroom with a celebrity!"
   front_join_us_learn_more: "Learn more"
 
+  cta_see_whats_new: "See what's new"
+  cta_try_more_activities: 'Try more activities'
```

Comment:
These are two new buttons for the Nov HoC banner on code.org - [see Creative brief](https://docs.google.com/document/d/12D64T4NrCY3E8mKmvn1baL9J3b8GkyXJfUAiLSnLZeA/edit#bookmark=id.vj07wqujnyn2)